Azure/azure-cli
# [Keyvault] `az keyvault private-endpoint-connection list`: Support list mhsm's private endpoint connections

**Description**

Feature requested by service team & Breadth coverage

**Testing Guide**

`az keyvault private-endpoint-connection list --hsm-name ystestmhsm`

---

This checklist is used to make sure that common guidelines for a pull request are followed.
